src/crate/crash/test_command.py

```diff
@@ -21,6 +22,23 @@
 from distutils.version import StrictVersion
 
 
+@contextmanager
+def crate_shell(hosts='localhost:4200', **kwargs):
+    """
+    Create an "auto-closable" CrateShell object that can be used in conjunction
+    with the ``with`` statement, for example::
+
+    >>> with crate_shell(error_trace=True) as cmd:
+    ...     # do something here
+
+    """
+    cmd = CrateShell(hosts.split(' '), **kwargs)
```

Comment:
I know this could be achieved by adding `__enter__` and `__exit__` methods to the CrateShell class, but I am reluctant to do so, as this is only for suppressing the warning in the test suite.